README.md

```diff
@@ -99,48 +99,7 @@ You can have the tool create a machine-readable log file by specifying the `--lo
   -y, --yaml=<yamlFile>                  The path to the yaml file. (Not needed for the get commands)
 ```
 
-##### [HINT - DEFAULT VALUES FOR SOME PARAMS]
 
-To reduce the number of the program parameters `api`, `organization` and `space`, there is a property file [cf_control.properties](cloud.foundry.cli/src/main/resources/cf_control.properties) that defines default values for the <code>get</code> command.
-For the <code>diff/apply</code> commands, the paramters can be fetched from the target section of the given YAML file.
-
-So the default values are defined in the cf_control.properties file:
-
-```
-  -a=api.run.pivotal.io
-  -o=cloud.foundry.cli
-  -s=development
-```
-the defined values in a given YAML file could be:
-
-```
-target:
-  endpoint=api.run.pivotal.io
-  org=cloud.foundry.cli
-  space=development
-```
```

Comment:
The part explaining the `target` section of a configuration should be reintroduced in the readme as soon as this feature is reimplemented in the controller classes and the `CfOperationsCreator` class.

Reply:
Yes, we will put it in, when we implement the feature #113 :)

Reply:
#113 has been merged weeks ago; which PR/issue do you really refer to? #131? In that case, may I suggest to reopen this issue?

Reply:
Not the issue 113, I mean the ID of the sprint backlog 113.

Reply:
I see. Makes sense. GitHub just created a link to the other issue/PR, so that confused me for a second. Funnily enough, if you change the order of the digits to 131, you actually end up in the original issue for the feature...